Frontend/src/app/views/dashboard/project-details/project-details.component.ts

```diff
@@ -254,6 +259,45 @@ export class ProjectDetailsComponent {
       this.toastrService.danger("There was an error updating the project", "Error");
       this.loadingProject = false;
     })
+    this.pageMode = 'view';
+  }
+
+  removeUserCheck(removeUserId: number) {
+    let title = "Are you sure you want to remove this user?";
+    let body = `Removing this user from the project will also remove all existing
+     time entries the user has created on this project.`;
+    this.dialogService.open(YesNoDialogComponent, {
+      context: {
+        body: body,
+        title: title,
+        yesClicked: () => {
+          this.removeUser(removeUserId)
+        }
+      }
+    });
+  }
+
+  removeUser(removeUserId: number) {
+    let removeUserInfo: ProjectRemoveUserDTO = {
+      projectId: this.details.id,
+      userId: removeUserId
+    }
+    this.projectService.removeUserFromProject(removeUserInfo).subscribe((res: GenericResponseDTO) => {
+      if (res.success) {
+        this.teamMembers.forEach((element, index) => {
+          if (removeUserId == element.data.id) {
+            this.teamMembers.splice(index, 1);
+          }
+          this.teamDataSource = this.dataSourceBuilder.create(this.teamMembers);
+        })
+        this.toastrService.show("Success", "User Id: " + res.data + " successfully removed from your project", { status: 'success', duration: 4000 });
+        this.userChangesMade = true;
```

Comment:
I think using the persons first name would be better here, the person using the application could care less about what the user ID is.